### PR TITLE
Upgrade all dependencies to latest

### PR DESCRIPTION
**Pull request by dtuite**
Upgrade all packages to latest.

Closes #220 
Closes #214

### CODE REVIEW

**Comment by dtuite** [comment]
https://app.clubhouse.io/larder/story/2483/update-marketing-site-dependencies

**Review by Irma12 — DISMISSED** [review]

**Comment by dtuite** [comment]
@Irma12 Thank you for the +1. I'm going to hold off merging this until after #226 is merged and then I might need to rebase it.

**Review by iain-b — DISMISSED** [review]

**Comment by dtuite** [comment]
@Irma12 Rebased. Could I get another +1 please?

**Review by Irma12 — APPROVED** [review]

**Pull request merged**
